ui/src/main/java/edu/wpi/grip/ui/pipeline/StepController.java

```diff
@@ -142,6 +191,72 @@ private void moveStepRight() {
     pipeline.moveStep(step, +1);
   }
 
+  @FXML
+  private void toggleExpand(MouseEvent event) {
+    if (event.getClickCount() == 1) {
+      expanded.set(!expanded.get());
+    } else if (event.getClickCount() == 2) {
+      eventBus.post(new SetStepsExpandedEvent(expanded.get()));
```

Comment:
I changed it so instead of double click all steps are now triggered with a secondary click.